packages/cache/src/resource-manager.ts

```diff
@@ -101,8 +101,14 @@ export class ResourceManager {
 
     if (item) {
       const code = await this.fetchCode(item.cachedUrl);
-      const uri = `data:text/javascript,${code};`;
-      return import(/* @vite-ignore */ uri) as Promise<T>;
+      if (code) {
+        // This if statement is needed to avoid an error when vite tries to inline an
```

Comment:
I think we can get rid of the comment because it seems quite reasonable to check if the code is defined before trying to import it, but should we throw an exception then or how should it be handled?

Reply:
fixed